setup.py

```diff
@@ -1,6 +1,7 @@
 from setuptools import setup
 
-from treelib import __version__
+__version__ = '1.6.1'
```

Comment:
moved `__version__` from `__init__` to `setup.py` because while trying to install dependencies in `setup.py` the imports would cause an error by importing `tree.Tree` class, and thus trying to import `future` at the beginning of module.

https://travis-ci.org/caesar0301/treelib/jobs/653746818?utm_medium=notification&utm_source=github_status